### ISSUE TITLE
OT refuses to open scenes, opens them if ffmpeg path is removed

### ISSUE THREAD

**Issue by agatre** [opentoonz/opentoonz#1153]
I wonder if opentoonz can't handle to open a scene again after it elements reach a certain point. It has happened twice, with the program crashing when trying to open a scene I was working on.

For some reason I removed the ffmpeg path, and it opened them. It should be noted that I had added videos for rotoscopy, and since ffmpeg was removed those particular files were gone. Are they the cause of the issue?

-Tried deleting the video file, added fmpeg path and reopen scene. Didn't work. Seems the latest version of ffmpeg causes some trouble with scenes.
-Tried updating ffmpeg with the binary from april, didn't work either. I wonder how I'll render my final work now.

**Comment by agatre** [comment]
This issue is even present in versions before the minimalist theme. I had the same problem which that update, which is why I changed to an older version. I'm on a Mac.

**Comment by turtletooth** [comment]
Can you give me some example files with steps to recreate the bug.  I'm guessing it is a ffmpeg timeout or a file name issue.

**Comment by Masonimous** [comment]
I've run into a similar problem, yet I didn't remove the ffmpeg or replace it. I used an audio file and a video file when I began working on my project. However, as I add more levels and columns to the project, OpenToonz ends up crashing more likely when I try to reopen the project. I know the situation is ffmpeg because when I open the project using OTX, when ffmpeg isn't detected, it opens fine. As soon as I link OTX with ffmpeg, it crashes just like OT. 

I was able to get it working with OT sometimes by opening the project with OTX and deleting the video file, saving the project, and reopening it in OT, but now I'm at a point where I don't have any more media to delete and ffmpeg cause the program to crash everytime I try to open the file.
